apps/i18n/common/en_us.json

```diff
@@ -12,10 +12,18 @@
   "abuseTypeOther": "Other",
   "abuseTypeQuestion": "How does this content violate the [Terms of Service]({url})?",
   "abusiveUrl": "URL of the content being reported",
-  "accountWelcomeBannerHeaderLabel": "Continue with a Code.org account",
-  "accountWelcomeBannerContentWorkshopEnroll": "To enroll in workshops, you’ll need a Code.org educator account.",
```

Comment:
These strings are still here, I just moved them because I realized they weren't in alphabetical order like the other strings